crate/operator/change_plan.py

```diff
@@ -0,0 +1,121 @@
+import logging
```

Comment:
I think we should avoid calling it "plan" anywhere as the operator has no such concept - as far as it's concerned, we're just changing the compute. How about `change_compute`, `CHANGE_COMPUTE` etc?

Reply:
Also need to align with @Taliik what the expected webhooks should be...